core/workspace_svg.js

```diff
@@ -2644,4 +2647,21 @@ WorkspaceSvg.prototype.getGrid = function() {
   return this.grid_;
 };
 
+/**
+ * Close tooltips, context menus, dropdown selections, etc.
+ * @param {boolean=} opt_onlyClosePopups Whether only popups should be closed.
+ */
+ WorkspaceSvg.prototype.hideChaff = function(opt_onlyClosePopups) {
+  Tooltip.hide();
+  WidgetDiv.hide();
+  DropDownDiv.hideWithoutAnimation();
+
+  var onlyClosePopups = !!opt_onlyClosePopups;
+  var autoHideables = this.getComponentManager().getComponents(
+      Blockly.ComponentManager.Capability.AUTOHIDEABLE, true);
+  autoHideables.forEach(function(autoHideable) {
+    autoHideable.autoHide(onlyClosePopups);
+  });
```

Comment:
[Per style guide]:

```suggestion
  autoHideables.forEach((autoHideable) => autoHideable.autoHide(onlyClosePopups));
```
(Get clang-format to fix line length!)

Reply:
Done